src/textures/Texture.js

```diff
@@ -53,9 +53,9 @@ class Texture extends EventDispatcher {
 		this.internalFormat = null;
 		this.type = type;
 
-		this.offset = new Vector2( 0, 0 );
```

Comment:
This was originally done to be more explicit and I don't think this needs refactoring.

TBH, I really against such PRs since even if the review effort is minor it still costs time. And these PR do not add a real value.

Reply:
Yes, you're right.

This place is more intuitive if it's explicit.

I just revised the PR https://github.com/mrdoob/three.js/pull/26775, by the way to see this, feel can delete...

Maybe we need to do something more worthwhile

To be honest, I'd like to do some feature development, but I don't know how to start...

Anyway, thank you for your patience and your review time.